packages/playground-preview-worker/src/index.ts

```diff
@@ -257,6 +264,14 @@ app.get(`${rootDomain}/api/inspector`, async (c) => {
 app.get(`${previewDomain}/.update-preview-token`, (c) => {
 	const url = new URL(c.req.url);
 	const token = url.searchParams.get("token");
+
+	if (
+		c.req.header("Sec-Fetch-Dest") !== "iframe" ||
```

Comment:
`sec-fetch-dest` is a pretty new header - only supported since Safari > 16.4 for example, released in March 2023.

This might exclude some older macOS/Safari versions from using the playground. The security implications probably outweigh that concern though.

Reply:
Good point but I think there are other reasons the playground doesn't work in Safari already